minisom.py

```diff
@@ -546,6 +547,16 @@ def test_activation_reponse(self):
     def test_activate(self):
         assert self.som.activate(5.0).argmin() == 13.0  # unravel(13) = (2,3)
 
+    def test_distance_from_weights(self):
+        self.som._weights = arange(2 * 3 * 4).reshape(2, 3, 4)
```

Comment:
do not overwrite the a variable filled in the setup. Also, `self.som` is supposed to have weights with shape `(5, 5, 1)`.